### PR TITLE
[OKU-22]Search bar enhancements

### PR DESCRIPTION
**Pull request by twentymurial33**
## Summary of changes

- The typed text should be black
- There's too much space on the left

## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-22/search-bar-enhancements

## Before & after screenshots or video recordings (Before & After)
Screen Shot 2023-03-08 at 11 08 43 AM

Screen Shot 2023-03-02 at 9 11 57 AM

Screen Shot 2023-03-09 at 1 34 03 PM

### CODE REVIEW

**Comment by idiglove** [comment]
Now we're updating the search bar, can we also ensure the search icon is positioned correctly? Forgot abt that, thanks :)

**Comment by twentymurial33** [comment]
> Now we're updating the search bar, can we also ensure the search icon is positioned correctly? Forgot abt that, thanks :)

Done

File: project_app/components/ToolBar.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ function ToolBar() {
...
               position: "absolute",
-              width: "18.99px",
-              height: "18.73px",
-              top: "170px",
-              left: "980px",
+              width: "28.99px",
+              height: "28.73px",
+              top: "168px",
+              left: "960px",
             }}
```
Comment (idiglove) [inline project_app/components/ToolBar.js:17]: can we put it in styled components pls?

Comment (twentymurial33) [reply project_app/components/ToolBar.js:17]: Yes done

File: project_app/components/ToolBar.js [hunk 2]
```diff
@@ -1,45 +1,23 @@
...
-      </div>
-    </Container>
+    <div>
+      <StyledContainer>
+        <input type="text" placeholder="Search board game" />
+        <section
+          style={{
+            top: "170px",
+            left: "970px",
+            position: "absolute",
```
Comment (idiglove) [inline project_app/components/ToolBar.js:13]: why do we still have this inline styles?

Comment (twentymurial33) [reply project_app/components/ToolBar.js]: sorry look at this commit @idiglove

**Review by idiglove — APPROVED** [review]
looks good! ⭐

**Pull request merged**